src/components/timepicker/timePicker.js

```diff
@@ -0,0 +1,53 @@
+import 'react-widgets/styles.css';
+import DatePicker from 'react-widgets/DatePicker';
+import TimeInput from 'react-widgets/TimeInput';
+
+import { useState } from 'react';
+
+function dateToEpoch (date) {
+    const timeInEpochSeconds = Date.parse(date)
+
+    // return epoch time in seconds
+    return timeInEpochSeconds / 1000
+}
+
+// Datetime input using 
+// React Widgets DatePicker and TimePicker
+function TimePicker(props) {
+    const [dateTime, setDateTime] = useState(new Date())
+    const [epochTime, setEpochTime] = useState(dateToEpoch(new Date()))
```

Comment:
When we create a date object, it'd be best to have the seconds default to 0, since we usually want the message to be posted on the exact minute.
```suggestion
    const currentDate = new Date();
    currentDate.setSeconds(0);

    const [dateTime, setDateTime] = useState(currentDate)
    const [epochTime, setEpochTime] = useState(dateToEpoch(currentDate))
```